dashboard/modules/job/job_manager.py

```diff
@@ -210,13 +210,22 @@ async def run(
                     "metadata": self._metadata,
                 }
             )
-            # Set RAY_ADDRESS to local Ray address, if it is not set.
+            # Always set RAY_ADDRESS as find_bootstrap_address address for
+            # job submission. In case of local development, prevent user from
+            # re-using http://{address}:{dashboard_port} to interact with
+            # jobs SDK.
+            # TODO:(mwtian) Check why "auto" does not work in entrypoint script
```

Comment:
@mwtian wait, is it not possible to use `ray.init(address="auto")` in a submitted job script? If so, that is a P0 bug

Reply:
I was not able to repro locally

Reply:
What I meant (for @jiaodong to add this TODO for me) is to investigate why overriding `RAY_ADDRESS` here to `auto`, instead of `find_bootstrap_address()`, does not work. `ray.init(address="auto")` should definitely work in the submitted job script.

Reply:
Thanks for the clarification, makes sense to me now